src/pages/send-tokens/hooks/use-send-form.ts

```diff
@@ -8,31 +9,30 @@ import { microStxToStx } from '@common/stacks-utils';
 import { FormValues } from '@pages/send-tokens/send-tokens';
 import { removeCommas } from '@common/token-utils';
 import { STX_TRANSFER_TX_SIZE_BYTES } from '@common/constants';
-import BigNumber from 'bignumber.js';
+import { stxTokenState } from '@store/assets/tokens';
 
 export function useSendAmountFieldActions({
   setFieldValue,
 }: Pick<FormikProps<FormValues>, 'setFieldValue'>) {
-  const balances = useFetchBalances();
+  const stxToken = useAtomValue(stxTokenState);
```

Comment:
you can create a hook for this, `useStxToken`, as i see you have the `useAtomValue` in multiple places. This is useful if the implementation behind this ever changes, you don't need to find and replace many atoms, etc.

Reply:
I see this already exists now in `use-assets`. There is a `useStxTokenState`.